case-lib/opt.sh

```diff
@@ -51,13 +51,15 @@ func_opt_parse_option()
             # option will accept parameter
             [ ${OPT_PARM_lst[$i]} -eq 1 ] && _short_opt=$_short_opt':' && _long_opt=$_long_opt':'
         done
-        _op_temp_script=`getopt -o "$_short_opt" --long "$_long_opt" -n "$ST_NAME" -- $@`
-        if [ $? != 0 ] ; then echo "Error parsing option" >&2 ; _func_opt_dump_help ; fi
+        _op_temp_script=$(getopt -o "$_short_opt" --long "$_long_opt" -- "$@") || {
+            echo "Error parsing option" >&2
```

Comment:
Would adding some data in the error message make it more useful and help fix mistakes faster?
```shell
>&2 printf 'Error parsing %s/%s/%s\n' "$i" "$short_opt" "$long_opt"# etc.
```